packages/ui/src/components/TextArea/index.tsx

```diff
@@ -0,0 +1,288 @@
+                id={id ?? localId}
+                autoFocus={autoFocus}
+                disabled={disabled}
+                rows={rows}
+                ref={ref}
+                value={value}
+                onChange={event => {
+                  onChange(event.currentTarget.value)
+                }}
+                isSuccess={!!computedSuccess}
+                isError={!!computedError}
+                isClearable={!!clearable}
+                minLength={minLength}
+                maxLength={maxLength}
+                placeholder={placeholder}
+                data-testid={dataTestId}
+                name={name}
+                onFocus={onFocus}
+                onBlur={onBlur}
+                readOnly={computedReadOnly}
+              />
+              <StyledTextAreaAbsoluteStack
+                direction="row"
+                alignItems="center"
+                gap="1"
+              >
+                {clearable ? (
+                  <Button
+                    aria-label="clear value"
+                    variant="ghost"
+                    size="xsmall"
+                    icon="close"
+                    onClick={() => {
+                      onChange('')
+                    }}
+                  />
+                ) : null}
+                {success ? (
+                  <Icon
+                    name="checkbox-circle-outline"
+                    color="success"
+                    size={STATE_ICON_SIZE}
+                  />
+                ) : null}
+                {error ? <Icon name="alert" color="danger" /> : null}
+              </StyledTextAreaAbsoluteStack>
+            </StyledTextAreaWrapper>
+          </Tooltip>
+
+          <Row templateColumns="minmax(0, 1fr) min-content" gap="1">
+            <div>
+              {computedSuccess ? (
+                <Text as="p" variant="caption" sentiment="success">
+                  {computedSuccess}
+                </Text>
+              ) : null}
+              {computedError ? (
+                <Text as="p" variant="caption" sentiment="danger">
+                  {computedError}
+                </Text>
+              ) : null}
+              {computedHelper && typeof computedHelper === 'string' ? (
+                <Text
+                  as="p"
+                  variant="caption"
+                  sentiment="neutral"
+                  prominence="weak"
+                  disabled={disabled}
+                >
+                  {computedHelper}
+                </Text>
+              ) : null}
+              {computedHelper && typeof computedHelper !== 'string'
+                ? computedHelper
+                : null}
+            </div>
+            {maxLength ? (
+              <Text
+                as="div"
+                sentiment="neutral"
+                prominence="weak"
+                variant="caption"
+              >
+                {value.length}/{maxLength}
+              </Text>
+            ) : null}
+          </Row>
```

Comment:
You need to condition this to be rendered or not, or we will have empty divs and spacing when not bottom text is displayed

<img width="2559" alt="Screenshot 2023-12-05 at 14 37 44" src="https://github.com/scaleway/ultraviolet/assets/15812968/13975c29-ddce-416f-9152-3dd29b2eb4b0">